fundrebalancer/fundrebalancer_test.go

```diff
@@ -668,14 +668,14 @@ func TestFundRebalancer_Rebalance(t *testing.T) {
 		mockDatabse := mock_database.NewMockDatabase(t)
 
 		mockEVMTxExecutor := evm2.NewMockEVMTxExecutor(t)
-		mockEVMTxExecutor.On("ExecuteTx", mockContext, arbitrumChainID, arbitrumAddress, []byte{}, "999", osmosisAddress, mock.Anything).Return("arbitrum hash", nil)
+		mockEVMTxExecutor.On("ExecuteTx", mockContext, arbitrumChainID, arbitrumAddress, []byte{}, "999", osmosisAddress, mock.Anything).Return("arbitrum hash", "", nil)
 
 		// mock executing the approval tx
-		mockEVMTxExecutor.On("ExecuteTx", mockContext, arbitrumChainID, arbitrumAddress, mock.Anything, "0", arbitrumUSDCDenom, mock.Anything).Return("arbitrum approval hash", nil)
+		mockEVMTxExecutor.On("ExecuteTx", mockContext, arbitrumChainID, arbitrumAddress, mock.Anything, "0", arbitrumUSDCDenom, mock.Anything).Return("arbitrum hash", "", nil)
 
 		mockDatabse.EXPECT().InsertSubmittedTx(mockContext, db.InsertSubmittedTxParams{
 			ChainID:  arbitrumChainID,
-			TxHash:   "arbitrum approval hash",
+			TxHash:   "arbitrum hash",
```

Comment:
i feel like this probably changed by mistake? if we don't have the hash as `arbitrum approval hash` then this would match when the actual rebalance tx is submitted which isn't what we want to test

Reply:
changed